src/ui/rideWindow.ts

```diff
@@ -21,10 +22,10 @@ export const model = new RideViewModel();
  * Window to edit properties of the currently selected ride.
  */
 export const rideWindow = window({
-	title: model._title,
+	title: "Edit ride",
 	position: "center",
-	width: { value: 233, min: 185, max: 250 },
-	height: 252,
+	width: 233, minWidth: 185, maxWidth: 250,
```

Comment:
This seems to be reverting to changes from an earlier version as well?

Reply:
Corrected it, I didn't update flexui before continuing in my own fork as well.